commit fb585cefd3648aa199ec2c12fd56440038c1b58c (refs/changes/75/875/2)
Author: Robert Mustacchi <rm@joyent.com>
Date:   2016-11-10T20:00:35+00:00 (2 years, 11 months ago)
    
    OS-5652 platform build needs to work with python 2.7 (fix pkgsrc)
    Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>
    Approved by: Patrick Mooney <patrick.mooney@joyent.com>

diff --git a/pkgsrc/onbld/packlist b/pkgsrc/onbld/packlist
index e42e6772..7697095d 100644
--- a/pkgsrc/onbld/packlist
+++ b/pkgsrc/onbld/packlist
@@ -84,47 +84,47 @@
 ./lib/perl/onbld_elfmod_vertype.pm
 ./lib/perl/onbld_elfmod.pm
 ./lib/python
-./lib/python2.6/onbld/__init__.py
-./lib/python2.6/onbld/__init__.pyc
-./lib/python2.6/onbld/Checks/__init__.py
-./lib/python2.6/onbld/Checks/__init__.pyc
-./lib/python2.6/onbld/Checks/Cddl.py
-./lib/python2.6/onbld/Checks/Cddl.pyc
-./lib/python2.6/onbld/Checks/CmtBlk.py
-./lib/python2.6/onbld/Checks/CmtBlk.pyc
-./lib/python2.6/onbld/Checks/Comments.py
-./lib/python2.6/onbld/Checks/Comments.pyc
-./lib/python2.6/onbld/Checks/Copyright.py
-./lib/python2.6/onbld/Checks/Copyright.pyc
-./lib/python2.6/onbld/Checks/CStyle.py
-./lib/python2.6/onbld/Checks/CStyle.pyc
-./lib/python2.6/onbld/Checks/DbLookups.py
-./lib/python2.6/onbld/Checks/DbLookups.pyc
-./lib/python2.6/onbld/Checks/HdrChk.py
-./lib/python2.6/onbld/Checks/HdrChk.pyc
-./lib/python2.6/onbld/Checks/JStyle.py
-./lib/python2.6/onbld/Checks/JStyle.pyc
-./lib/python2.6/onbld/Checks/Keywords.py
-./lib/python2.6/onbld/Checks/Keywords.pyc
-./lib/python2.6/onbld/Checks/ManLint.py
-./lib/python2.6/onbld/Checks/ManLint.pyc
-./lib/python2.6/onbld/Checks/Mapfile.py
-./lib/python2.6/onbld/Checks/Mapfile.pyc
-./lib/python2.6/onbld/Checks/ProcessCheck.py
-./lib/python2.6/onbld/Checks/ProcessCheck.pyc
-./lib/python2.6/onbld/hgext/__init__.py
-./lib/python2.6/onbld/hgext/__init__.pyc
-./lib/python2.6/onbld/hgext/cdm.py
-./lib/python2.6/onbld/Scm/__init__.py
-./lib/python2.6/onbld/Scm/__init__.pyc
-./lib/python2.6/onbld/Scm/Backup.py
-./lib/python2.6/onbld/Scm/Backup.pyc
-./lib/python2.6/onbld/Scm/Ignore.py
-./lib/python2.6/onbld/Scm/Ignore.pyc
-./lib/python2.6/onbld/Scm/Version.py
-./lib/python2.6/onbld/Scm/Version.pyc
-./lib/python2.6/onbld/Scm/WorkSpace.py
-./lib/python2.6/onbld/Scm/WorkSpace.pyc
+./lib/python2.7/onbld/__init__.py
+./lib/python2.7/onbld/__init__.pyc
+./lib/python2.7/onbld/Checks/__init__.py
+./lib/python2.7/onbld/Checks/__init__.pyc
+./lib/python2.7/onbld/Checks/Cddl.py
+./lib/python2.7/onbld/Checks/Cddl.pyc
+./lib/python2.7/onbld/Checks/CmtBlk.py
+./lib/python2.7/onbld/Checks/CmtBlk.pyc
+./lib/python2.7/onbld/Checks/Comments.py
+./lib/python2.7/onbld/Checks/Comments.pyc
+./lib/python2.7/onbld/Checks/Copyright.py
+./lib/python2.7/onbld/Checks/Copyright.pyc
+./lib/python2.7/onbld/Checks/CStyle.py
+./lib/python2.7/onbld/Checks/CStyle.pyc
+./lib/python2.7/onbld/Checks/DbLookups.py
+./lib/python2.7/onbld/Checks/DbLookups.pyc
+./lib/python2.7/onbld/Checks/HdrChk.py
+./lib/python2.7/onbld/Checks/HdrChk.pyc
+./lib/python2.7/onbld/Checks/JStyle.py
+./lib/python2.7/onbld/Checks/JStyle.pyc
+./lib/python2.7/onbld/Checks/Keywords.py
+./lib/python2.7/onbld/Checks/Keywords.pyc
+./lib/python2.7/onbld/Checks/ManLint.py
+./lib/python2.7/onbld/Checks/ManLint.pyc
+./lib/python2.7/onbld/Checks/Mapfile.py
+./lib/python2.7/onbld/Checks/Mapfile.pyc
+./lib/python2.7/onbld/Checks/ProcessCheck.py
+./lib/python2.7/onbld/Checks/ProcessCheck.pyc
+./lib/python2.7/onbld/hgext/__init__.py
+./lib/python2.7/onbld/hgext/__init__.pyc
+./lib/python2.7/onbld/hgext/cdm.py
+./lib/python2.7/onbld/Scm/__init__.py
+./lib/python2.7/onbld/Scm/__init__.pyc
+./lib/python2.7/onbld/Scm/Backup.py
+./lib/python2.7/onbld/Scm/Backup.pyc
+./lib/python2.7/onbld/Scm/Ignore.py
+./lib/python2.7/onbld/Scm/Ignore.pyc
+./lib/python2.7/onbld/Scm/Version.py
+./lib/python2.7/onbld/Scm/Version.pyc
+./lib/python2.7/onbld/Scm/WorkSpace.py
+./lib/python2.7/onbld/Scm/WorkSpace.pyc
 ./man/man1onbld/bldenv.1onbld
 ./man/man1onbld/bringovercheck.1onbld
 ./man/man1onbld/cddlchk.1onbld
